Unix/tests/wsman/utils.cpp

```diff
@@ -308,6 +309,9 @@ int StartServerAndConnect(
 
     UT_ASSERT( MI_RESULT_OK == result );
 
+    // A new socket connection is created, give it a little time
+    ut::sleep_ms(200);
```

Comment:
This sort of thing is true for other hard-coded sleep statements above this too ...